crates/symbolicator/src/services/cficaches.rs

```diff
@@ -299,7 +299,6 @@ impl CfiCacheActor {
 #[tracing::instrument(skip_all)]
 fn write_cficache(path: &Path, object_handle: &ObjectHandle) -> Result<(), CfiCacheError> {
     configure_scope(|scope| {
-        scope.set_transaction(Some("compute_cficache"));
```

Comment:
in theory it seems to me this should have been some kind of nested transaction inside the main request one?  does the product handle nested transactions?

Reply:
so i'm curious then why you decided to remove these transactions?

Reply:
because they would overwrite the `symbolicate_stacktraces` name.